### PR TITLE
From MicronautStream 8.1 to EclipseStore 1.0.0

### PR DESCRIPTION
**Pull request by sdelamo**
Fixes: https://github.com/micronaut-projects/micronaut-microstream/issues/299

> This is the first release of EclipseStore, the successor of [MicroStream](https://github.com/microstream-one). It is at the same state as the last MicroStream Storage release, [8.1.1](https://github.com/microstream-one/microstream/releases). Just living in a different namespace.

Cache no longer available. 

dependency mappings:
https://github.com/eclipse-store/store/blob/main/storage/embedded-tools/storage-migrator/src/main/resources/META-INF/mappings/dependency.mappings

package mappings:
https://github.com/eclipse-store/store/blob/main/storage/embedded-tools/storage-migrator/src/main/resources/META-INF/mappings/package.mappings

### CODE REVIEW

File: src/main/docs/guide/whatsnew.adoc [hunk 1]
```diff
@@ -1 +1 @@
-Since Micronaut MicroStream 2, you can use <<storageTargets, Storage Targets>> such as PostgreSQL, Amazon S3 and DynamoDB.
+Since Micronaut EclipseStore 2, you can use <<storageTargets, Storage Targets>> such as PostgreSQL, Amazon S3 and DynamoDB.
```
Comment (wetted) [inline src/main/docs/guide/whatsnew.adoc:1]: This doesn't seem right. There is no EclipseStore 2. I think the line can be deleted.

File: src/main/docs/guide/toc.yml
Comment (wetted) [inline src/main/docs/guide/toc.yml:1]: I don't think you need the What's New section (yet).

**Review by wetted — CHANGES_REQUESTED** [review]
Looks good. Just the docs What's New needs to be addressed (i.e. deleted).

File: src/main/docs/guide/toc.yml
Comment (sdelamo) [reply src/main/docs/guide/toc.yml:1]: resolved via https://github.com/micronaut-projects/micronaut-eclipsestore/pull/1/commits/2ee181a999c0b7fa8cda54f87af54f9ece8c369a

File: src/main/docs/guide/whatsnew.adoc (hunk 1 above)
Comment (sdelamo) [reply src/main/docs/guide/whatsnew.adoc]: resolved via https://github.com/micronaut-projects/micronaut-eclipsestore/pull/1/commits/2ee181a999c0b7fa8cda54f87af54f9ece8c369a

**Review by wetted — APPROVED** [review]

**Pull request merged**